doc/nrf/protocols/bt/bt_mesh/configuring.rst

```diff
@@ -246,3 +246,44 @@ Using the :ref:`bluetooth_mesh_sensor_server` sample as an example, configured a
+  * :kconfig:option:`CONFIG_BT_MESH_USES_TFM_PSA` - Enables use of the TF-M PSA API based security toolbox (default option for platforms that support TF-M).
+  * :kconfig:option:`CONFIG_BT_MESH_USES_TINYCRYPT` - Enables use of Tinycrypt-based security toolbox.
+    Zephyr's Mesh operates with open value of keys, including storing them in the persistent memory.
+    The Tinycrypt-based solution has worse security materials protection compared to others and not recommended for future designs.
+
+The Bluetooth Mesh security toolbox based on the PSA API does not operate with open key values.
+After Bluetooth Mesh gets open key value, it immediately imports the keys into the crypto library and receives the unique key identifier.
+The key identifiers are used in the security toolbox and stored in the persistent memory.
+The crypto library is responsible for storing of the open key values in the Internal Trusted Storage (ITS).
+Bluetooth Mesh data structures based on Tinycrypt and based on the PSA API as well as images of these structures stored in the persistent memory are not compatible due to different key representations.
+When a provisioned device updates its image using the Tinycrypt-based toolbox on image with the PSA API based toolbox, it needs to be unprovisioned first and reprovisioned after the update.
+If the image is changed over Mesh DFU, it is recommended to use :c:enumerator:`BT_MESH_DFU_EFFECT_UNPROV`.
+
+However, a provisioned device can update its image using the Tinycrypt-based toolbox on image with the PSA API based toolbox without being unprovisioned.
+The :kconfig:option:`CONFIG_BT_MESH_KEY_IMPORTER` Kconfig option enables the key importer functionality.
+The key importer is an application initialization functionality that is called with kernel initialization priority before starting main.
+This functionality reads out the persistently stored Bluetooth Mesh data and if it finds keys stored by the Tinycrypt-based security toolbox, it imports them over the PSA API into the crypto library and stores the key identifiers in PSA API toolbox based format.
+When the application starts Bluetooth Mesh initialization, the persistent area already has the stored data in the correct format.
+
+Using the key importer might make the device vulnerable for attacks.
+If the device works with the key importer functionality enabled and the attacker writes arbitrary data in the persistent memory, fake keys might be stored and imported to PSA crypto library after the next device reset.
+
+Complete the following steps to use the key importer functionality safely:
+
+1. Update the images with the Tinycrypt-based toolbox and the PSA API based toolbox with the key importer feature enabled.
+#. Reset the device to perform the key import after the devices in the network have successfully updated their images.
+#. Update the images with the PSA API based security toolbox with the key importer feature disabled.
+
```

Comment:
I'm confused here. 
- Why one should "reset" the device? Does reset mean a factory reset or a reboot? 
- Opening line says: "Complete following ... to use key importer ...", and second bullet says disable key importer?

Reply:
Yes, it should be reboot. (I'll fix it)
Finally, key importer should be disabled since it adds vulnerability. It should be used only once to import keys and then it is not required at all.